src/cdi_ndi/config.h.in

```diff
@@ -23,8 +23,8 @@
 
 /* NDI version (if found) */
 #cmakedefine NDI_VERSION_STRING "@NDI_VERSION@"
-#cmakedefine NDI_MAJOR "@NDI_MAJOR@"
-#cmakedefine NDI_MINOR "@NDI_MINOR@"
+#cmakedefine NDI_MAJOR @NDI_MAJOR@
```

Comment:
@KineticTheory OK with you if these become integers? I couldn't figure out a way to convert strings to integers for comparison in the `NDI_DEDX_SUPPORT` macro

Reply:
This is fine.